### ISSUE TITLE
Hybrid model accuracy not improving even after 150 iteration with training sample size of 1000 images.

### ISSUE THREAD

**Issue by 108mk** [darthsimpus/ASOC#2]
Hi @darthsimpus ,
I am following the instruction as per the details mentioned in the paper and this repo.

the output expected from files: [ClassicalSNNGenerator.ipynb] and [data_create_KMNIST.ipynb] are reproduced successfully.

But, with [[5_ary]_pretrain_5x2_squeeze_KMNIST], I am unable to achieve the accuracy of the hybrid model beyond 50% even after 150 iterations and 1000 training sample sizes.

the classical pre-trained model achieves a good accuracy of up to 99% on  KMNIST (as per file: [ClassicalSNNGenerator.ipynb] )

I would appreciate it if you could help me with this issue.

**Comment by dikoling** [comment]
I'm in the same situation as yours
